utils/checkExists.js

```diff
@@ -0,0 +1,16 @@
+const db = require('../db/connection');
+const format = require('pg-format');
+
+module.exports = checkExists
```

Comment:
Generally module.exports should be at the bottom of a file, because of hoisting this does work but it looks a little strange seeing the function being exported first and then declared after.